src/index.ts

```diff
@@ -181,6 +210,24 @@ namespace Private {
   /**
    * A counter for unique IDs.
    */
-  export
   let id = 0;
+
```

Comment:
Why don't we need to export this anymore?

Reply:
It is no longer used outside of this namespace, so does not need to be exported.